src/WebJobs.Script/Workers/Http/DefaultHttpWorkerService.cs

```diff
@@ -170,5 +172,40 @@ private void AddRequestHeadersAndSetRequestUri(HttpRequestMessage httpRequestMes
             httpRequestMessage.Headers.Add(HttpWorkerConstants.HostVersionHeaderName, ScriptHost.Version);
             httpRequestMessage.Headers.UserAgent.ParseAdd($"{HttpWorkerConstants.UserAgentHeaderValue}/{ScriptHost.Version}");
         }
+
+        public async Task<bool> IsWorkerReady()
+        {
+            await Utility.DelayAsync(WorkerConstants.WorkerInitTimeoutSeconds, WorkerConstants.PollingIntervalMilliseconds, async () =>
```

Comment:
Should the polling interval here have a more descriptive name? I'm assuming we won't be sharing this with other actions against the worker, correct?